docs/requirements.txt

```diff
@@ -1 +1,2 @@
 sphinxcontrib-fulltoc
+.
```

Comment:
This tells readthedocs to install all requirements for `linode_api4` (polling, etc.)